ibexOverview.js

```diff
@@ -0,0 +1,140 @@
+var PORT = 60000;
+var HOST = "http://dataweb.isis.rl.ac.uk" 
+//$('.btn').button();
+
+function refresh() {
+	$.ajax({
+		url: HOST + ":" + PORT + "/",
+		dataType: 'jsonp',
+		data: {"Instrument": "all"},
+		timeout: 1000,
+		error: function(xhr, status, error){ 
+			window.alert("Error: JSON not read");
+		},
+		success: function(data){ 
+			//window.alert(JSON.stringify(data));
+			display_data(data);
+		}
+	});
+}
+
+function clearBox(elementID){
+    document.getElementById(elementID).innerHTML = "";
+};
+
+function sortObject(o) {
+    var sorted = {},
+    key, a = [];
+
+    for (key in o) {
+        if (o.hasOwnProperty(key)) {
+            a.push(key);
+        };
+    };
+
+    a.sort();
+
+    for (key = 0; key < a.length; key++) {
+        sorted[a[key]] = o[a[key]];
+    }
+    return sorted;
+}
+
+function close_window(){
+	clearBox("new_window");
+};
+
+function create_new_window(){
+	clearBox("new_window");
+	var newWindow = document.createElement("div");
+	var newWindowStyle = document.createAttribute("class");
+	newWindowStyle.value = "col-sm-12 well";
+	newWindow.setAttributeNode(newWindowStyle);
+	newWindow.id = "internal";
+	document.getElementById("new_window").appendChild(newWindow);
+	var closeButton = document.createElement("button");
+	closeButton.innerHTML = "x";
+	var blockListStyle = document.createAttribute("onclick");
+    blockListStyle.value = "close_window()";
+	closeButton.setAttributeNode(blockListStyle);
+	document.getElementById("internal").appendChild(closeButton);	
+};
+
+function open_wall_display(){
+	create_new_window()
+	var newIframe = document.createElement("iframe");
+	newIframe.src = "http://epics-jenkins.isis.rl.ac.uk/plugin/jenkinswalldisplay/walldisplay.html?viewName=All&jenkinsUrl=http%3A%2F%2Fepics-jenkins.isis.rl.ac.uk%2F"
+	newIframe.height = String(windowHeight*2/5)+"px"
+	newIframe.width = "100%"
+	//newIframe.frameborder = "0"
+	document.getElementById("internal").appendChild(newIframe);
+	
+	
+}
+
+
+function on_click(elmnt) {
+	create_new_window();
+	var newIframe = document.createElement("iframe");
+	newIframe.src = "http://dataweb.isis.rl.ac.uk/IbexDataweb/default.html?Instrument="+ elmnt.innerHTML
+	newIframe.height = String(windowHeight*2/5)+"px"
```

Comment:
- Define magic numbers (i.e. `0.4`) as variables
- Use `0.4` rather than `2/5` to avoid possibility of bad integer arithmetic. Also just more natural from a coding perspective